test/tests/api/v2_0/os_install_tests.py

```diff
@@ -46,8 +50,75 @@ def teardown(self):
     def __get_data(self):
         return loads(self.__client.last_response.data)
 
+    def __get_compute_nodes(self):
+        Api().nodes_get_all()
+        nodes = self.__get_data()
+        compute_nodes = []
+        for n in nodes:
+            type = n.get('type')
+            if type == 'compute':
+                compute_nodes.append(n)
+        LOG.info('compute nodes count {0}'.format(len(compute_nodes)))
+        return compute_nodes
+
+    def __wait_for_completion(self, node, graph_name, graph_instance):
+        id = node.get('id')
+        start_time = date.datetime.now()
+        current_time = date.datetime.now()
+        while True:
+            if (current_time - start_time).total_seconds() > DEFAULT_TIMEOUT_SEC:
+                raise TypeError('Timed out after {0} seconds'.format(DEFAULT_TIMEOUT_SEC))
```

Comment:
This is not TypeError (see TypeError definition https://docs.python.org/2/library/exceptions.html#exceptions.TypeError), please choose another proper Error type or even a generic Error will be better than TypeError.